### PR TITLE
hippocratic oath and product collaboration

### PR DESCRIPTION
**Pull request by mmmries**
I wanted to write up a few notes along the lines of our recent team discussions about how we can collaborate with product to provide feedback on what gets built at Spiff

### CODE REVIEW

File: 04.CheckingIn.md [hunk 1]
```diff
@@ -18,3 +18,22 @@ The more important metrics are the ones you bring to the discussion.
 Share the things you want to measure about your own performance (see [Setting Your Course](02.SettingYourCourse.md)).
 Do you both agree that these are worthwhile?
 If you agree on the value of your goals, spend some time discussin what will be measured, how you will measure it and how you plan to impact those metrics.
+
+## Having a Voice
+
+Product and Engineering should alway shave a collaborative relationship.
```
Comment (casejamesc) [inline 04.CheckingIn.md:24]: `always`

File: 03.GettingWorkDone.md [hunk 2]
```diff
@@ -42,3 +42,28 @@ This is exactly why we want human beings focused on these questions and why we p
...
+* Does no harm to users?
+  * Does this change compromise the privacy of our users or customers?
+  * Does this change make it harder to get things done for our users?
+  * Will users be surprised or confused by these changes?
+* Does no harm to our platform?
+  * Will this change cause unstable memory usage? Does it have a query that will cause a problems for the production database?
+  * When this feature breaks, what will the error message look like? How hard will it be to troubleshoot the problem?
+  * Should we track any metrics around these changes? Do we need observability tools?
+* Does no harm to the project?
+  * Is this code understandable for a new engineering 3 months from now?
```
Comment (casejamesc) [inline 03.GettingWorkDone.md]: `engineer`

File: 04.CheckingIn.md (hunk 1 above)
Comment (casejamesc) [inline 04.CheckingIn.md]: `always have`

File: 03.GettingWorkDone.md [hunk 3]
```diff
@@ -42,3 +42,28 @@ This is exactly why we want human beings focused on these questions and why we p
 
 Projects at Spiff are generally setup to automatically deploy when we merge to master.
 The project README should specify if this is not the case.
+
+## The Hippocratic Oath of the Shipit
+
+When you are reviewing code from a teammate, it's your responsibility to protect and increase the quality of our system.
+Much like the [Hippocratic Oath](https://en.wikipedia.org/wiki/Hippocratic_Oath), we need to "first do no harm".
+Specificaly you should consider the following types of harm:
```
Comment (casejamesc) [inline 03.GettingWorkDone.md]: `Specifically`

File: 03.GettingWorkDone.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ They are also a great time to ask questions.
 > "I followed the pattern in UpdateUserMutation here, does anyone know why we do it this way?"
 
 During code review we try to automate as much of the mundane feedback as possible.
-Code formatting standards should be enforced by CI so that we aren't paying human beings to argue about it.
```
Comment (camkidman) [inline 03.GettingWorkDone.md:29]: 👍

**Review by casejamesc — APPROVED** [review]
Really cool stuff @mmmries. This is great to start developing these materials 👍.

**Pull request merged**